app/uk/gov/hmrc/customs/declaration/model/models.scala

```diff
@@ -18,9 +18,13 @@ package uk.gov.hmrc.customs.declaration.model
 
 import java.util.UUID
 
+import scala.xml.Elem
+
 case class RequestedVersion(versionNumber: String, configPrefix: Option[String])
 
```

Comment:
The reason I originally overrided `toString` was for value classes `ConversationId` and `CorrelationId` that take `UUID` - an exceptional case. For value classes that wrap a string we can just use `.value` - otherwise it can be confusing in the code to know if a `toString` transformation is in play

Reply:
:+1: